packages/commonwealth/server/scripts/purgeCloudflareCache.ts

```diff
@@ -0,0 +1,41 @@
+import { logger } from '@hicommonwealth/core';
+import fetch from 'node-fetch';
+import { config } from '../config';
+
+const log = logger(import.meta);
+
+async function purgeCache(zoneId?: string, apiKey?: string) {
+  if (!zoneId || !apiKey) {
+    log.warn('Missing Cloudflare env variables. Skipping cache purge');
+    return;
+  }
+
+  const url = `https://api.cloudflare.com/client/v4/zones/${zoneId}/purge_cache`;
+  const body = {
+    purge_everything: true,
+  };
+  const headers = {
+    Authorization: `Bearer ${apiKey}`,
+    'Content-Type': 'application/json',
+  };
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: headers,
+      body: JSON.stringify(body),
+    });
+
+    const responseData = await response.json();
+    console.log('Cache purge request successful:');
+    console.log(responseData);
+  } catch (error) {
+    console.error('Error purging cache:', error.message);
```

Comment:
throw?

Reply:
Definitely log this with pino so we get a Rollbar warning.